lesson_5/index5.html

```diff
@@ -23,10 +24,14 @@ <h1>Lesson_4</h1>
                 </td>
             </tr>
             <tr>
-                <td><b>Bill to</b><p>Street adress</p>
-                    <p>City, state, zip</p></td>
-                <td><b>Ship to</b><p>Street adress</p>
-                    <p>City, state, zip</p></td>
+                <td><b>Bill to</b>
```

Comment:
it should be from a new line, like this:

```
<td>
  <b>Bill to</b>
  <p>Street adress</p>
  City, state, zip
</td>
```

Reply:
ok